doc_build/toc.md.tpl

```diff
@@ -0,0 +1,29 @@
+# rules_pkg - {VERSION}
+
+<div class="toc">
+  <h2>Common Attributes</h2>
+    <ul>
+      <li><a href="#common">Package attributes</a></li>
+      <li><a href="#mapping-attrs">File attributes</a></li>
+    <ul>
+
+  <h2>Packaging Rules</h2>
+  <ul>
+    <li><a href="#pkg_deb">pkg_deb</a></li>
+    <li><a href="#pkg_tar">pkg_tar</a></li>
+    <li><a href="#pkg_rpm">pkg_rpm</a></li>
+    <li><a href="#pkg_zip">pkg_zip</a></li>
+  </ul>
+
+  <h2>File Tree Creation Rules</h2>
+  <ul>
+    <li><a href="#filter_directory">filter_directory</a>
+    <li><a href="#pkg_filegroup">pkg_filegroup</a>
+    <li><a href="#pkg_files">pkg_files</a>
+    <li><a href="#pkg_mkdirs">pkg_mkdirs</a>
+    <li><a href="#pkg_mklink">pkg_mklink</a>
+    <li><a href="#pkg_attributes">pkg_attributes</a>
+    <li><a href="#strip_prefix.files_only">strip_prefix</a>
```

Comment:
Huh.  Looks like the `strip_prefix` pseudomodule isn't itself documented.  Could you create a task for doing that?

Reply:
#482